frontend/src/components/Collections/components/Grid/components/Row/CollectionRow/index.tsx

```diff
@@ -75,17 +81,20 @@ const CollectionRow: FC<Props> = (props) => {
   } = aggregateDatasetsMetadata(datasets);
 
   return (
-    <StyledRow>
+    <StyledCollectionRow
+      onClick={handleRowClick}
+      data-test-class="collection-row"
```

Comment:
I haven't found the need to have both `data-test-id` and `data-test-class`, especially since I think `data-test-class` doesn't support css class selector `.class-name` (e.g., `page.$$('.collection-row')` 🤔 So perhaps if we really need it, we could use attribute `className="collection-row"` instead?

Reply:
Sounds good! But I don't think `data-test-id` supports css class selector `#id-name` either

Reply:
https://playwright.dev/docs/selectors/#id-data-testid-data-test-id-data-test-selectors

Reply:
Yeah that's correct! Which is why we have to use `getTestTag()` helper function 👍